### DISCUSSION TITLE
Docker images for armv7

### DISCUSSION THREAD

**Discussion by JimScope** [uxbox/uxbox#1821] [Q&A] [answered]
I was checking that there are no more Docker images in armv7. Will they recreate them in future versions of penpot or does it no longer support armv7?

**Comment by niwinz** [comment, marked as answer]
We have not generated the armv7 images because we think that they are not useful. But it is pretty easy to add them, so if they are useful for you, we will add them to the pipeline.

**Reply by JimScope** [reply]
Thanks a lot for the answer

**Reply by niwinz** [reply]
as you can observe, the latest images are already build with armv7 architecture
